sass/site/site.scss

```diff
@@ -158,7 +158,7 @@ $paddingNormal: 20px;
 // --------------------------------------------------
 .navbar {
   margin-bottom: 0px;
-  border-top: 3px solid $blueDark;
+  border-top: 3px solid #0d94d2; //cerner blue
```

Comment:
Do we not have Cerner Blue and Cerner green saved as variables somewhere? This is Sass after all.